src/styles/blocks/head-block/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  padding: 0 50px;
+  background-color: $background_white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  @media (max-width:1024px) {
+    & {
+      max-width: 1024px;
+    }
+
+    .nav--position {
+      margin-left: 243px;
```

Comment:
Avoid using magic number for margin value. Consider using a SCSS variable for this value, which could be more descriptive and reusable across the project.